From 2330fa34ec0c6163aa8f47f5d75a36589234715b Mon Sep 17 00:00:00 2001
From: David King <amigadave@amigadave.com>
Date: Mon, 15 Jun 2015 05:51:25 +0100
Subject: [PATCH] Fix crash when failing to read FLAC headers

Ensure that the GError passed to g_file_read() has been initialized.

https://bugzilla.redhat.com/show_bug.cgi?id=1231502
---
 src/tags/flac_header.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tags/flac_header.c b/src/tags/flac_header.c
index f2c1109..fee66b8 100644
--- a/src/tags/flac_header.c
+++ b/src/tags/flac_header.c
@@ -60,7 +60,7 @@ et_flac_header_read_file_info (GFile *file,
         return FALSE;
     }
 
-    istream = g_file_read (file, NULL, &state.error);
+    istream = g_file_read (file, NULL, error);
 
     if (istream == NULL)
     {
-- 
2.4.3

